openedx_lti_tool_plugin/http.py

```diff
@@ -0,0 +1,24 @@
+"""HTTP objects for openedx_lti_tool_plugin."""
+import logging
+
+from django.http import HttpResponseBadRequest
+
+log = logging.getLogger(__name__)
+
+
+class LoggedHttpResponseBadRequest(HttpResponseBadRequest):
+    """A HTTP 400 response class that sends the response to the log."""
+
+    def __init__(self, *args, **kwargs):
+        """HTTP response __init__ method.
+
+        Args:
+            *args: Variable length argument list.
+            **kwargs: Arbitrary keyword arguments.
+        """
+        super().__init__(*args, **kwargs)
+
+        try:
+            log.error(args[0])
+        except IndexError:
```

Comment:
```suggestion
    def __init__(self, message, *args, **kwargs):
        """HTTP response __init__ method.

        Args:
            *args: Variable length argument list.
            **kwargs: Arbitrary keyword arguments.
        """
        super().__init__(message, *args, **kwargs)

        log.error(message)
```

What I meant with the comment was this